Enchancement: Notifications indicator when loading is not finished

Raroun opened the pull request:
I have been working on the loading indicator. 
At first, I wanted a spinning loading indicator, but that wasn't possible, at least not for me.

If notifications have not yet been loaded, “loading...” is now displayed.
Tested on friendica.opensocial.space in live operation with both the Frio and Vier themes.
Fixes #15097

Commit:
Added notofication indicators

Commit:
ensure loading is visible by default

Commit:
fixed javascript event timing

Commit:
updated frio template which uses it´s own nav.tpl

Commit:
handle both selector patterns

Commit:
fix for subtle "loading..." - make it more present

Commit:
preserve oading and empty state elements when rebuilding the menu

Commit:
forget the non working spinner in the navigation menu

Commit:
fix for overwriting nav-update event

Commit:
simpler approach

Commit:
added some "padding"

marcusxss commented:
Oh, thanks for looking into this!

I haven't taken an in depth look, but I do have a few suggestions:
1. The indentation seems to be off in some places?
2. Maybe squash the commits down to one or a few commits?
3. Is the change from `$('#topbar-first').bind` to `$(document).bind` necessary? I believe the change means it has to traverse a much larger part of the document to find the relevant tag, so it's slower?
4. The CI pipeline complains about code standards. It can be fixed by running `php-cs-fixer`.

Commit:
fix: revert jQuery bind from  back to  for better performance

The change from .bind to .bind causes performance
issues as it has to traverse a much larger DOM scope. Reverting back to the
more specific selector for better performance.
